TombEngine/Scripting/Internal/TEN/Objects/ObjectsHandler.h

```diff
@@ -64,6 +64,45 @@ class ObjectsHandler : public ScriptInterfaceObjectsHandler
 			return std::make_unique<R>(std::get<R::IdentifierType>(m_nameMap.at(name)));
 	}
 
+	template <typename R, char const* S>
+	std::vector <std::unique_ptr<R>> GetMoveablesBySlot(GAME_OBJECT_ID objID)
+	{
+		std::vector<std::unique_ptr<R>> items = {};
+		for (auto& [key, val] : m_nameMap)
+		{
+			if (!std::holds_alternative<std::reference_wrapper<MESH_INFO>>(val))
```

Comment:
Mentioned this in discord but putting it here too, you want to test for `std::holds_alternative<short>(val)`, otherwise you will get things like AI objects and cameras.

Reply:
done